### PR TITLE
Feature/ko docs

### PR DESCRIPTION
**Pull request by Rojojun**
Thank you for your contributions! 👍

### CODE REVIEW

**Comment by Rojojun** [comment]
* Improving translation style
* Korean-friendly approach

File: docs/ko/_coverpage.md
Comment (adamlui) [inline docs/ko/_coverpage.md:1]: @Rojojun in README.md you translated "Importing the library" (which the Get Started button should link to) as:

> 라이브러리 호출

However, here it is translated as:

> 라이브러리를 불러오는 중

So you should change it in README.md or here to match each other

File: docs/ko/README.md
Comment (adamlui) [inline docs/ko/README.md:1]: @Rojojun for the Install/Readme/Discuss link labels, can you translate them too?

**Review by adamlui — CHANGES_REQUESTED** [review]

File: docs/ko/README.md
Comment (Rojojun) [reply docs/ko/README.md:1]: Okay!! I got it 😄

File: docs/ko/_coverpage.md
Comment (Rojojun) [reply docs/ko/_coverpage.md:1]: Thank you for your comment! I will fix it!

**Review by adamlui — APPROVED** [review]
Nice thanks for the translations!

**Pull request merged**